test/performance/locustfile.py

```diff
@@ -0,0 +1,58 @@
+import urllib3, os, json
```

Comment:
I believe we should include the license header also in the python files:

```suggestion
# Copyright KubeArchive Authors
# SPDX-License-Identifier: Apache-2.0

import urllib3, os, json
```

Reply:
Added

Reply:
I don't see the header...

Reply:
Added